tools/scxtop/src/perfetto_trace.rs

```diff
@@ -14,12 +14,10 @@ use std::collections::BTreeMap;
 use std::fs;
 use std::time::{SystemTime, UNIX_EPOCH};
 
-use crate::bpf_skel::types::bpf_event;
-use crate::edm::{ActionHandler, BpfEventHandler};
+use crate::edm::ActionHandler;
 use crate::{
     Action, CpuhpEnterAction, CpuhpExitAction, ExecAction, ExitAction, ForkAction, GpuMemAction,
-    HwPressureAction, IPIAction, SchedSwitchAction, SchedWakeupAction, SchedWakingAction,
```

Comment:
Should probably add some action handlers for some of these events, since they could be added to the traces. I think some of them are unhandled.

Reply:
Downside of these all being on one line, I think it's only `HwPressureAction` that's unused.